metrics/mauve/README.md

```diff
@@ -89,10 +90,20 @@ This metric outputs a dictionary with 5 key-value pairs:
 
 ### Values from popular papers
 
-The [original MAUVE paper](https://arxiv.org/abs/2102.01454) reported values ranging from 0.88 to 0.94 for open-ended text generation using a text completion task in the web text domain. The authors found that bigger models resulted in higher MAUVE scores, and that MAUVE is correlated with human judgments.
+The [original MAUVE paper](https://arxiv.org/abs/2102.01454) reported values ranging from 0.88 to 0.94 for open-ended text generation using a text completion task in the web text domain. The authors found that bigger models resulted in higher MAUVE scores and that MAUVE is correlated with human judgments.
+
+### Best practices
```

Comment:
This is not a valid section for the README (you can see the template [here](https://github.com/huggingface/evaluate/blob/main/templates/%7B%7B%20cookiecutter.module_slug%20%7D%7D/README.md)).

It would most likely go under 'Limitations and Bias'